runtime/src/dice.rs

```diff
@@ -101,6 +116,10 @@ fn cert_from_dccm(
                 .get(..persistent_data.fht.fmcalias_tbs_size.into()),
             dv.fmc_dice_signature(),
         ),
+        CertType::RtAlias => (
+            persistent_data.rtalias_tbs.get(..RTALIAS_TBS_SIZE as usize),
```

Comment:
We should probably add rt alias size to the FHT in this PR. Otherwise we are generating the cert incorrectly, because RTALIAS_TBS_SIZE is the size of the whole DCCM region, not just the TBS portion of the cert.

Reply:
Done